components/Layout/Layout.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Header from '../Header';
+import StyledButton from '../elements/Button';
+
+const Layout = () => (
+  <>
+    <Header hasBear={false} />
```

Comment:
It would be better to make the `hasBear` an optional prop and set it to false on default